cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLConfig.java

```diff
@@ -246,6 +250,7 @@ public String toString() {
    */
   @SuppressWarnings("unchecked")
   public abstract static class Builder<T extends Builder> {
+    protected List<Object> comments;
```

Comment:
Why use `List<Object>` for comments? What can go inside a comment beside string?

Reply:
Line 60 uses List<Object>, trying to keep it consistent